cpp/src/solver/shuffle.h

```diff
@@ -26,8 +26,6 @@
 namespace ML {
 namespace Solver {
 
-using namespace MLCommon;
```

Comment:
Is this related or just a random cleanup?

Reply:
Related because it introduces an unnecessary `using namespace` in the new files that include this file, and creates unnecessary issues. Also a cleanup because it is not used or needed